packages/release/release.spec

```diff
@@ -122,6 +122,7 @@ Requires: %{_cross_os}kexec-tools
 Requires: %{_cross_os}keyutils
 Requires: %{_cross_os}makedumpfile
 Requires: %{_cross_os}netdog
+Requires: %{_cross_os}nvme-cli
```

Comment:
nit: we can omit the dependency in "release" since it will come in via `ghostdog`